dashboard/modules/serve/serve_agent.py

```diff
@@ -211,6 +212,7 @@ async def put_all_deployments(self, req: Request) -> Response:
 
         try:
             client.deploy_apps(config)
+            record_extra_usage_tag(TagKey.SERVE_REST_API_VERSION, "v1")
```

Comment:
is this possible we add it in the client.py? In this case, we can also gather information by kuberay & anyscale service.

Reply:
KubeRay and other service providers all use this REST API, so this should still capture their usage. I'd prefer to track this here because the metric gets updated directly whenever the REST API route is called successfully.